<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0" colspan="11">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_y_offset">
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="minimum">
       <double>-10.000000000000000</double>
      </property>
      <property name="maximum">
       <double>10.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.050000000000000</double>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="11">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_temp_offset">
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="minimum">
       <double>-100.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.500000000000000</double>
      </property>
     </widget>
    </item>
    <item row="13" column="11">
     <widget class="QPushButton" name="btn_import">
      <property name="text">
       <string>Import</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="11">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_z_offset">
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="minimum">
       <double>-10.000000000000000</double>
      </property>
      <property name="maximum">
       <double>10.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.050000000000000</double>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="11">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>temperature offset</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>x offset</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>z offset</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Import file</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Device</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="11">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_x_offset">
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="minimum">
       <double>-10.000000000000000</double>
      </property>
      <property name="maximum">
       <double>10.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.050000000000000</double>
      </property>
     </widget>
    </item>
    <item row="0" column="10">
     <widget class="QLineEdit" name="lineEdit_import_file"/>
    </item>
    <item row="1" column="10">
     <widget class="QComboBox" name="comboBox_device"/>
    </item>
    <item row="0" column="11">
     <widget class="QPushButton" name="btn_browse">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>y offset</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="10" column="4">
     <widget class="QCheckBox" name="checkBox_3">
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="10" column="9">
     <widget class="QCheckBox" name="checkBox_8">
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="10" column="2">
     <widget class="QCheckBox" name="checkBox_2">
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="10" column="5">
     <widget class="QCheckBox" name="checkBox_4">
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="10" column="7">
     <widget class="QCheckBox" name="checkBox_6">
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="10" column="6">
     <widget class="QCheckBox" name="checkBox_5">
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="10" column="8">
     <widget class="QCheckBox" name="checkBox_7">
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QCheckBox" name="checkBox_9">
      <property name="text">
       <string>A</string>
      </property>
     </widget>
    </item>
    <item row="11" column="2">
     <widget class="QCheckBox" name="checkBox_10">
      <property name="text">
       <string>B</string>
      </property>
     </widget>
    </item>
    <item row="11" column="4">
     <widget class="QCheckBox" name="checkBox_11">
      <property name="text">
       <string>C</string>
      </property>
     </widget>
    </item>
    <item row="11" column="5">
     <widget class="QCheckBox" name="checkBox_12">
      <property name="text">
       <string>D</string>
      </property>
     </widget>
    </item>
    <item row="11" column="6">
     <widget class="QCheckBox" name="checkBox_13">
      <property name="text">
       <string>E</string>
      </property>
     </widget>
    </item>
    <item row="11" column="7">
     <widget class="QCheckBox" name="checkBox_14">
      <property name="text">
       <string>F</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
